shell_sort/shell_sort.py

```diff
@@ -0,0 +1,37 @@
+def shell_sort(array):
+    """
+    Sorts the given array of integers using the Shell Sort algorithm
+    """
```

Comment:
@Enether Recently we have added `PEP8Bear` and `PyLintBear` which checks for these issues, that is why other python codes don't have it, but we'll be fixing it soon.